src/ngraph/ngraph_compiler.cc

```diff
@@ -400,7 +400,7 @@ void Compiler::DeepCopy(const nnvm::Graph& graph) {
 // Check nodes in NGraph
 void Compiler::CheckInNgraph() {
   std::unordered_set<std::string> unsupported_op_names;
-  for (auto node : ngraph_.nodes_) {
+  for (const std::shared_ptr<ngraph_bridge::Node>& node : ngraph_.nodes_) {
```

Comment:
I apologize for the verbosity.  I made this change because we have several (at least 3?) "node" classes that are relevant here.  So I wanted to be more explicit about which one `node` refers to.  Without that I was getting confused.